integration/src/test/scala/ammonite/integration/BasicTests.scala

```diff
@@ -122,6 +122,12 @@ object BasicTests extends TestSuite{
       val evaled = exec('basic / "Resources.sc")
       assert(evaled.out.string.contains("1745"))
     }
+    'testSilentIvy{
+      val evaled1 = exec('basic/"scalaTags.sc")
+      assert(evaled1.err.string.contains("scalatags"))
+      val evaled2 = exec('basic/"scalaTags.sc", "-s")
+      assert(!evaled2.err.string.contains("scalatags"))
+    }
```

Comment:
This test doesn't seem rigorous enough; it's not obvious to me what it is testing.